plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoMetadata.java

```diff
@@ -15,6 +15,7 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
```

Comment:
Could you please add a test to `TestMongoConnectorTest`? e.g. `testProjectionPushdown`

Reply:
Do we have similar tests for other connectors? I believe that this is covered by existing tests